sns_ik_examples/src/ik_tests.cpp

```diff
@@ -225,12 +257,18 @@ void test(ros::NodeHandle& nh, double num_samples_pos, double num_samples_vel, s
     if (int((double)i/num_samples_pos*100)%10 == 0)
       ROS_INFO_STREAM_THROTTLE(1,int((i)/num_samples_pos*100)<<"\% done");
   }
-  ROS_INFO_STREAM("SNS-IK found "<<success<<" solutions ("<<100.0*success/num_samples_pos<<"\%) with an average of "<<total_time/num_samples_pos<<" secs per sample");
-
+  vst.score=100.0*success/num_samples_pos;
+  vst.avg_time=total_time/num_samples_pos;
+  ROS_INFO_STREAM(vst.name<<" found "<<success<<" solutions ("<<vst.score<<"\%) with an average of "<<vst.avg_time<<" secs per sample");
```

Comment:
Include the success with scaling results, or I could add that after this merge. Also, watch your line length....